README.md

```diff
@@ -1,15 +1,27 @@
-# Amaze File Manager
+# <a href="https://teamamaze.xyz">Amaze File Manager</a>
+
+<img height="50px" src="icon.png" data-canonical-src="icon.png" >
+
 ![CI](https://github.com/TeamAmaze/AmazeFileManager/workflows/Android%20Main%20CI/badge.svg?branch=master)
 [![Codacy Badge](https://api.codacy.com/project/badge/Grade/9ea2667dabaa4e8c98dbf0876ebacd3e)](https://app.codacy.com/gh/TeamAmaze/AmazeFileManager?utm_source=github.com&utm_medium=referral&utm_content=TeamAmaze/AmazeFileManager&utm_campaign=Badge_Grade_Settings)
 [![Codacy Badge](https://app.codacy.com/project/badge/Coverage/50d8e153feba47b9a8ff82ff57274c56)](https://www.codacy.com/gh/TeamAmaze/AmazeFileManager?utm_source=github.com&utm_medium=referral&utm_content=TeamAmaze/AmazeFileManager&utm_campaign=Badge_Coverage)
-[![GitHub release](https://img.shields.io/github/release/TeamAmaze/AmazeFileManager.svg)](https://github.com/TeamAmaze/AmazeFileManager/releases)
-[![License: GPL v3](https://img.shields.io/badge/License-GPL%20v3-blue.svg)](https://www.gnu.org/licenses/gpl-3.0)
+[![GitHub release](https://img.shields.io/github/release/TeamAmaze/AmazeFileManager.svg)](https://github.com/TeamAmaze/AmazeFileManager/releases)  
+[![Chat on Telegram](https://img.shields.io/badge/Telegram-2CA5E0?style=for-the-badge&logo=telegram&logoColor=white)](https://t.me/joinchat/NNFgWxQUB8fFE3yCsC0w4g)
+[![XDA Developers](https://img.shields.io/badge/XDA-Developers%20-%23AC6E2F.svg?&style=for-the-badge&logo=XDA-Developers&logoColor=white)](http://forum.xda-developers.com/android/apps-games/app-amaze-file-managermaterial-theme-t2937314)
+
```

Comment:
Badges will look better if they are on a single line, IMO (or the ones with the same size on the same line)

Reply:
Like this?
![imagen](https://user-images.githubusercontent.com/10991116/114062081-7e22d000-986d-11eb-9373-748d10610aa1.png)

Reply:
I still have no idea how to fix the lower AFH badge.

Reply:
I meant those small badges, BTW :) But this looks cool too :D

Reply:
Oh! The other badges are in different lines because they are for different things, the first is state of the project and the second line is social nets.